content/en/getting_started/tagging/unified_service_tagging.md

```diff
@@ -75,11 +75,122 @@ To setup unified service tagging in a containerized environment:
 
 #### Configuration
 
+##### Automatic version tagging for containerized environments
+You can use the `version` tag to [monitor deployments][17] and to identify faulty code deployments through [Automatic Faulty Deployment Detection][18].
+
+Datadog sets the `version` tag for you in the following priority order. If you manually set `version`, Datadog does not override your `version` value.
+
+| Priority         | Version Value |
+|--------------|------------|
+| 1    |  {your version value}       |
+| 2   | {image_tag}_{git_commit_sha}       |
+| 3         |  {image_tag} or {git_commit_sha} if only one is available      |
+
+You need to install Datadog Agent Version 7.52.0 or greater and enable Git in the tracer to fully enable automatic version tagging. You can learn how to enable Git in the tracer by reading [Embed Git information in your build artifacts][19] 
```

Comment:
Maybe we can spell this out more clearly? The git integration is recommended but not absolutely required.

Requirements:
- Datadog Agent Version 7.52.0
- If running containerized and `image_tag` suffices, no further configuration necessary
- If not running containerized or if you'd also like to have the git SHA included, (embed Git information...)

Then again the simplest direction might be best.

Reply:
Updated